shielddash/core/static/js/app/components/views/fetch-error.js

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 export default function(props) {
   return (
     <dl className="component-error-message">
-      <dt>{props.status}</dt>
+      <dt>{props.status} - {props.detail}</dt>
```

Comment:
To elaborate further we should still use the detail and say something like "The server came back with: 'Token used too late'"